src/common/fivegconfig.cc

```diff
@@ -0,0 +1,303 @@
+// Copyright (c) 2018-2022, Rice University
+// RENEW OPEN SOURCE LICENSE: http://renew-wireless.org/license
+
+/**
+ * @file fivegconfig.cc
+ * @brief Implementation file for the 5G configuration class which which imports
+ * json configuration values into class variables and verifies that the 
+ * specified configuration is compatible with 5G standards
+ */
+
+#include "fivegconfig.h"
+#include "logger.h"
+#include "utils.h"
+
+using json = nlohmann::json;
+
+static constexpr size_t kSubframesPerFrame = 10;
+static constexpr size_t kFlexibleSlotFormatIdx = 2;
+static constexpr size_t kMaxSlotFormat = 55;
+static constexpr bool kDebug = false;
+
+std::map<size_t, std::string> FiveGConfig::format_table {};
```

Comment:
Is there a case where you need to access this if there are no members of the class?   static is ok, but what are the drawbacks of making it a member function?  How many instances of this class will you be making?